sdks/python/apache_beam/runners/dataflow/dataflow_metrics.py

```diff
@@ -145,7 +145,7 @@ def _populate_metric_results(self, response):
 
     # Now we create the MetricResult elements.
     result = []
-    for metric_key, metric in metrics_by_name.iteritems():
+    for metric_key, metric in metrics_by_name.items():
```

Comment:
So in some places you seem to use `six.iteritems` it might be better to be consistent about that? There probably aren't that many metrics though so this isn't a big deal.